lib/ts/recipe/thirdparty/utils.ts

```diff
@@ -135,6 +164,7 @@ function validateAndNormaliseEmailVerificationConfig(
     config?: TypeInput
 ): TypeNormalisedInputEmailVerification {
     return {
+        emailDelivery: config?.emailDelivery,
```

Comment:
This is not correct right? It should be undefined (since we pass the emailDelivery instance directly when creating a new email verification instance).

Also, how is the types for this not complaining here? it was complaining in emailpassword right? weird..

Reply:
I see.